src/main/java/Model/Person.java

```diff
@@ -0,0 +1,37 @@
+package Model;
+
+import javax.persistence.*;
+
+@Entity
+public abstract class Person {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Integer id = null;
+    private String name = null;
+
+    public Person() {}
+
+    public Person(String name){
+        this.name = name;
+    }
+
+    public Integer getId(){
+        return this.id;
+    }
+
+    public String getName(){
+        return this.name;
+    }
+
+    public void setName(String name){
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return "Person [id=" + id + ", name=" + name + "]";
```

Comment:
we probably don't need this to be implemented since it's an abstract class and will never be instantiated